service/frontend/namespace_handler.go

```diff
@@ -893,7 +903,7 @@ func (d *namespaceHandlerImpl) maybeUpdateFailoverHistory(
 	if l := len(namespaceDetail.ReplicationConfig.FailoverHistory); l > 0 {
 		lastFailoverVersion = namespaceDetail.ReplicationConfig.FailoverHistory[l-1].FailoverVersion
 	}
-	if lastFailoverVersion != newFailoverVersion {
```

Comment:
should we just check if the lastFailoverVersion != oldFailoverVersion?

Reply:
actually, the impl of namespace failover history is confusing
1. the history is not replicated
2. when will the `lastFailoverVersion == newFailoverVersion`? this is the original check?

Reply:
lastFailoverVersion should always equal to oldFailoverVersion?